tribomobile/__tests__/App-test.js

```diff
@@ -1,14 +1,27 @@
 /**
  * @format
  */
-
 import 'react-native';
 import React from 'react';
 import App from '../App';
+import colors from '../src/colors';
 
-// Note: test renderer must be required after react-native.
-import renderer from 'react-test-renderer';
+const FixedDimensionsBasics = () => {
+  return (
+    <View>
+      <View
+        style={{width: 50, height: 50, backgroundColor: colors.GreenContent}}
```

Comment:
The inline styles are not recommended. Could you move the lines where you use inline styles to the stylesheets object?